app.js

```diff
@@ -2,6 +2,7 @@
 
 var async = require('async');
 var checkIpInList = require('./helpers/checkIpInList.js');
+var parametersReader = require('./helpers/parametersReader.js');
```

Comment:
`parametersReader` looks like bad name for var and file. This is only used for generating some list from command line args. But maybe I'm wrong, just more like a feeling. :)